### PR TITLE
Fix hueUtils.dexss

### PR DESCRIPTION
**Pull request by penggongkui**
#1547 
how about fix by this.

### CODE REVIEW

**Comment by romainr** [comment]
:+1: , thanks!

Nits: just you might have an extra space, could also do add another test with `false` instead of `0` too

File: desktop/core/src/desktop/js/utils/hueUtils.ts [hunk 1]
```diff
@@ -303,7 +303,7 @@ export const logError = (error: unknown): void => {
 export const equalIgnoreCase = (a?: string, b?: string): boolean =>
   !!a && !!b && a.toLowerCase() === b.toLowerCase();
 
-export const deXSS = (str?: string): string => (str && sanitizeHtml(str)) || '';
+export const deXSS = (str?: string): string => (str !== undefined) ? sanitizeHtml(str) : '';
```
Comment (romainr) [inline desktop/core/src/desktop/js/utils/hueUtils.ts:306]: nit: ?
(str !== undefined) --> str !== undefined

**Comment by romainr** [comment]
I also see @JohanAhlen just sent https://github.com/cloudera/hue/pull/1549 that has more type checks

**Comment by penggongkui** [comment]
ok, I will close this PR

**Comment by penggongkui** [comment]
#1549 same PR, close

**Comment by JohanAhlen** [comment]
Haha, seems like we fixed it in parallel. Thanks for finding the root cause!

**Pull request closed without merging**